### DISCUSSION TITLE
ESP32-C2

### DISCUSSION THREAD

**Discussion by milo1987** [micropython/micropython#13700] [ESP32] [answered]
Does anyone got a ESP32-C2 Devkit (with ESP8684) working with micropython?
The Ports on Github are online for C3 und does not work with the C2.
Or does anyone has a hint for me, how to compile for this board?

Thx for your Response

**Comment by peterhinch** [comment]
I don't think a C2 port has been done yet.

**Comment by projectgus** [comment, marked as answer]
It hasn't merged yet, but support for ESP32-C2 has been submitted in this pull request:
https://github.com/micropython/micropython/pull/15906

(Currently you need to make a custom build from this pull request to use it, but after it's merged then the upcoming nightly snapshot builds should include ESP32C2 firmware images and then the next MicroPython release will also include it.)
